lib/ansible/modules/net_tools/cloudflare_dns.py

```diff
@@ -811,8 +842,8 @@ def main():
         result, changed = cf_api.ensure_dns_record()
         if isinstance(result, list):
             module.exit_json(changed=changed, result={'record': result[0]})
-        else:
-            module.exit_json(changed=changed, result={'record': result})
+
```

Comment:
This seems like a change to the logic, rather than a docs update. What does this change signify?

Reply:
Nothing, it's something that PEP8 recommends.

There's no impact.